app/presenters/formats/edition_format_presenter.rb

```diff
@@ -0,0 +1,83 @@
+module Formats
+  class EditionFormatPresenter
+    def initialize(edition)
+      @edition = edition
+      @artefact = edition.artefact
+    end
+
+    def render_for_publishing_api(republish: false)
+      {
+        title: @edition.title,
+        base_path: base_path,
+        description: @edition.overview || "",
+        schema_name: schema_name,
+        document_type: @artefact.kind,
+        need_ids: [],
+        public_updated_at: public_updated_at,
+        publishing_app: "publisher",
+        rendering_app: "frontend",
+        routes: routes,
+        redirects: [],
+        update_type: update_type(republish),
+        change_note: @edition.latest_change_note,
+        details: details,
+        locale: @artefact.language,
+      }
+    end
+
+  private
+
+    def external_related_links
+      @edition.artefact.external_links.map do |link|
+        {
+          url: link["url"],
+          title: link["title"]
+        }
+      end
+    end
+
+    def routes
+      [
+        { path: "#{base_path}", type: path_type },
+        { path: "#{json_path}", type: "exact" }
```

Comment:
These json requests redirect to the content api in [api_redirectable.rb](https://github.com/alphagov/frontend/blob/master/app/controllers/concerns/api_redirectable.rb) which is included in all the format controllers. But yeah, probably makes sense to redirect to the content store api instead... we'll update Frontend to redirect to that as we migrate I reckon